flow/packet-headers/rsvp.yaml

```diff
@@ -0,0 +1,1222 @@
+      description: >-
+        Class = RECORD_ROUTE, Type1 ROUTE_RECORD C-Type = 1
+        Subobject: Label, C-Type: 3
+      type: object
+      properties:
+        length:
+          description: >-
+              The Length contains the total length of the subobject in bytes, including the Type and Length fields.  
+              The Length MUST be atleast 4, and MUST be a multiple of 4.
+          $ref: '#/components/schemas/Flow.RSVPRouteRecord.Length'
+          x-field-uid: 1
+        flags:
+          x-field-pattern:
+            description: >-
+              0x01 = Global label. This flag indicates that the label will be understood if received on any interface.
+            format: integer
+            length: 8
+            default: 1
+          x-field-uid: 2 
+        c_type:
+          x-field-pattern:
+            description: >-
+              The C-Type of the included Label Object. Copied from the Label object.
+            format: integer
+            length: 8
+            default: 0
+          x-field-uid: 3
+        contents_of_label_obejct:
+          description: >-
+            The contents of the Label Object. Copied from the Label Object.
+          type: string
+          format: hex
+          default: "00"
+          x-field-uid: 4
+    Flow.RSVPRouteRecord.Length:
+      type: object
+      properties:
+        choice:
+          description: auto or configured value.
+          type: string
+          default: auto
+          x-field-uid: 1
+          x-enum:
+            auto:
+              x-field-uid: 1
+            value:
+              x-field-uid: 2
+        auto:
+          description: >-
+            The OTG implementation can provide a system generated value for this property.
+            If the OTG is unable to generate a value the default value must be used.
+          type: integer
+          format: uint32
+          maximum: 256
+          default: 8
+          x-field-uid: 2
+        value:
+          type: integer
+          format: uint32
+          maximum: 256
+          default: 8
+          x-field-uid: 3 
+    Flow.RSVP.PathObjectsCustom:
+      type: object
+      description: Custom packet header
+      required: [bytes]
+      properties:
+        type:
+          x-field-pattern:
+            description: >-
+              User defined object type.
+            format: integer
+            length: 8
+            default: 0
+            features: [count]
+          x-field-uid: 1
+        length:
+          $ref: '#/components/schemas/Flow.RSVP.ObjectOptionsCustom.Length'
+          x-field-uid: 2
+        bytes:
+          description: >-
+            A custom packet header defined as a string of hex bytes.
+            The string MUST contain sequence of valid hex bytes.
+            Spaces or colons can be part of the bytes but will be discarded.
+            This packet header can be used in multiple places in the packet.
+          type: string
```

Comment:
There should be a max length ... Can we use hex type you mentioned ?

Reply:
Yes. Changed it to hex type and added max length.
MaxLenth = 131050 (65525 * 2 hex character per byte)
Maximum 65528 bytes can be added as object-contents in RSVP header.
3 bytes for type and length, so rest is 65525.